### PR TITLE
V1.5.2

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 1.5.2
- Set base numpy version to 1.23 for py311.

Links:
home: https://pandas.pydata.org/
dev: https://github.com/pandas-dev/pandas/tree/v1.5.2
release note: https://pandas.pydata.org/pandas-docs/version/1.5.2/whatsnew/index.html
installation doc: https://pandas.pydata.org/pandas-docs/version/1.5.2/getting_started/install.html#installation
setup: https://github.com/pandas-dev/pandas/blob/v1.5.2/setup.cfg
Jira: https://anaconda.atlassian.net/browse/PKG-811

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
Might need to add `pytest-cov` in the test section.

https://github.com/pandas-dev/pandas/blob/v1.5.2/requirements-dev.txt#L6


Might need to add `pytest-asyncio >=0.17` on the test section.

https://github.com/pandas-dev/pandas/blob/v1.5.2/requirements-dev.txt#L9

`psutil` seems to be mentioned as a requirement
https://github.com/pandas-dev/pandas/blob/v1.5.2/requirements-dev.txt#L8

**Comment by cbouss** [comment]
> https://github.com/pandas-dev/pandas/blob/v1.5.2/requirements-dev.txt#L6

The dependencies to look at should really be:
- Build deps: https://github.com/pandas-dev/pandas/blob/v1.5.2/pyproject.toml#L4
- Run deps: https://github.com/pandas-dev/pandas/blob/v1.5.2/setup.cfg#L34
- Run constrained deps: https://pandas.pydata.org/pandas-docs/version/1.5.2/getting_started/install.html#optional-dependencies

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
-{% set version = "1.5.1" %}
+{% set version = "1.5.2" %}
 {% set numpy = "1.20.3" %}  # [py<310 and (not osx and arm64)]
-{% set numpy = "1.21" %}  # [py>=310 or (osx and arm64)]
+{% set numpy = "1.21" %}  # [py==310 or (osx and arm64)]
```
Comment (ELundby45) [inline recipe/meta.yaml:3]: Just to clarify, `osx-arm` is fine with numpy 1.21 on py311?

**Review by ELundby45 — APPROVED** [review]
LGTM, Just one question.

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:3]: It's not but it will be overridden by the next line.

**Review by katietz — APPROVED** [review]
thanks

**Pull request merged**